assemble/conf/accumulo-site.xml

```diff
@@ -39,6 +39,15 @@
     <name>tserver.memory.maps.native.enabled</name>
     <value>true</value>
   </property>
+  <!-- Used by the PerTableVolumeChooser which is the default general.volume.chooser -->
```

Comment:
The big question that remains is since the PerTableVolumeChooser is the default, then the system will not work until the table and scoped volume choosers are set.  Hence I added them here in the example accumulo-site.xml and configured them in the mini accumulo configuration.